### DISCUSSION TITLE
Horror Dashboard Ui

### DISCUSSION THREAD

**Discussion by mitraxis** [mirumee/saleor-dashboard#4186] [General]
Hi. What happend with that lovely Ui? 
The new dashboard has zero contrast, tiny typography and menus. 
Everything looks like an Excel stylsheet.

Why do you think this is apealing to customers? 
When I presented the new dashboard to the client the first thing they said "boy that is ugly and small..."
That's not what you want to hear in a Pitch.

**Comment by mirekm** [comment]
Thank you for your feedback, Muamer.

At Saleor, our primary focus with the recent dashboard redesign has been to enhance productivity, especially when working with large data sets and complex e-commerce operations. While we understand the significance of aesthetics, we prioritize functionality to ensure our users can achieve their tasks with efficiency. For instance, the new datagrids in the product view enable swift price updates in both regular and full-screen modes without a lot of back and forth with the variant details view.

That said, we recognize the importance and value of a balanced design system. Pushing boundaries of what is known as a standard e-commerce admin is nuanced (and we optimize for productivity for global e-commerce operators). The current UI is in a transitionary phase, and we're actively working on improving elements like contrast and typography.

Let's maybe set up a quick call with our team to understand the feedback better.

**Reply by mitraxis** [reply]
Hi Mirek, Thank for the reply. 
I understand what you and your team aimed for, but it's getting a very negative reaction in my sales pitches so I thouth to give you my thouths on it.

Sure, let me know when you have time for a call. I'll gladly provide more feedback.

I'll send you my email on LinkedIn.
